OctopusDSC/OctopusDSCHelpers.ps1

```diff
@@ -96,7 +96,15 @@ function Write-Log {
 }
 
 function Invoke-OctopusServerCommand ($arguments) {
-    Write-Verbose "Executing command '$octopusServerExePath $($arguments -join ' ')'"
+    if
```

Comment:
It would be better to still print the command, just with the values obscured. Otherwise its confusing, as some commands are printed, some are not.

Reply:
Tricky, but not impossible. I can think of a way to do it, but the tests would have to change. Might merge this one now and work on that next.

Reply:
OK, I've got an idea how to suppress the --password and --masterkey command line params. And another method to suppress a password in a SQL connection string. Not sure how to test those yet though.

I'll merge this, then start a new branch to test those out, and start a new issue to track progress.